test/e2e/test/default.go

```diff
@@ -31,8 +38,12 @@ func DefaultSecurityContext() *corev1.PodSecurityContext {
 // with special permissions is created by APM test's builder
 // so that this can work.
 func APMDefaultSecurityContext() *corev1.PodSecurityContext {
```

Comment:
nit: Maybe we should update the godoc for this function as it is no longer accurate.